### ISSUE TITLE
Incremental container deployment

### ISSUE THREAD

**Issue by hipponix** [ansible/ansible-container#885]
##### ISSUE TYPE

 - Feature Idea

##### SUMMARY

Not sure it exists already as I went through the docs and examples and haven't find it yet (I might be wrong though).
It would be nice to be capable of adding a new container to an existing and running PROD (or DEV) environment without running **ansible-container destroy** and then **ansible-container build**.

I imagine a situation where we have a running microservices infrastructure containing few containers and need to deploy another one or more and not to touch the running ones.

As far as i can see, if you need to achieve that then the "destroy" command should be used and followed by "build" again. We would end up destroying / re-creating the infrastructure just because we need an additonal container.

In short, it would be nice to have a command which reads the current configuration and based on that it adds only those containers which are not running or haven't created yet.

If something already exists or I have misunderstood the ansible-container concept, I apologise for wasting your time. If not, then i will be more than happy to help on the new feature.

Best,
Mirco

```

```

```

```

**Comment by j00bar** [comment]
Why are you using `destroy` when you probably just mean `stop`?

**Comment by hipponix** [comment]
closed
